src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
+  text-align: center;
+}
```

Comment:
- [ ] Most styles from this CSS file now can be deleted since these styles were used by the boilerplate component generated by create-react-app. We delete any class or style that's not going to be used